en/config/battery.md

```diff
@@ -0,0 +1,81 @@
+If you cannot find any label on your battery telling you the number of cells you can infer the count by the battery type and its nominal voltage.
+
+So for li-po batteries the following voltages are possible
+* 1S - 3.7V
+* 2S - 7.4V
+* 3S - 11.1V
+* 4S - 14.8V
+* 5S - 18.5V
+* 6S - 22.2V
+
+This setting sets the number of cells connected in series PX4 should monitor.
+
+### Full cell voltage
+A chemical battery cells output voltage depends on many parameters like the way the battery was built, temperature, load, if the battery was damaged. But for a healthy battery under normal conditions with no load attached the remaining capacity is the main factor.
+Normal li-po batteries should be charged by dedicated charging devices such that each cell reaches a maximal voltage of exactly 4.2V. But after the battery was fully charged and is stored for later use the voltage starts to decrease even with no external load at all.
+That's why this setting should be set just a bit lower than the maximal voltage of a cell otherwise the fuel estimate will likely already be below 100% by the time you connect the battery to your autopilot.
+
+### Empty cell voltage
+Most battery cells particularly also li-pos suffer a lot when they get used after they are "empty". To say if they are empty is not binary and not accurately possible by only knowing the cell voltage.
+On one hand if a li-po ever sees a voltage lower than 3.2V even if it's under load it will suffer significant damage for sure. On the other hand the nominal cell voltage 3.7V without load is where the battery starts to get into the empty range and that's the most conservative voltage to consider it empty.
+In between these extremes the earlier you put the battery to charge again the longer the life span will be and the less capacity degrade you'll see.
+
+> **Hint** As my personal most general rule of thumb if the first li-po cell raw voltage gets below 3.5V during flight (under load) I start to land immediately.
+
+When speaking about the empty cell voltage we need to distinguish:
+* Raw voltage under load, the voltage directly measured while flying
+This value highly varies depending on the momentary current draw from the cells. As a result you will see fluctuating battery percentages
+* Rest voltage without a load when the battery is resting for some time
+Best suitable measurement to infer charge status of the cell.
+* Load compensated voltage calculated in air (under load)
+Ideally you want to have a load compensated voltage available during flight which reflects the rest voltage without load. but in practise the load compensated value will always be a bit below what you end up with because the battery "recovers" its unloaded voltage over time without gaining usable capacity.
+
+## Without load compensation
```

Comment:
This is a  level 2 heading, but appears to/may belong to the preceding heading/discussion on empty cell voltage. If it doesn't belong to that section then it is "floating" because at this point I don't know what "without load compensation" applies to.

Reply:
You can run the estimation without load compensation which is default, easiest to configure, worse performance compared to the other two methods. If you only set the "Basic Battery Settings" which are currently available directly in the Power UI of QGC you don't have load compensation.

I'm sorry if I structured it confusing, you basically have the following options:
- only set your cell count
you get a coarse percentage estimation based on the voltage measurement which highly fluctuates depenting on load
- reduce the fluctuation with load balancing
you get a lot less offset while hovering and fluctuation but it's still voltage based
- cobine the voltage based estimate with the accumulated used capacity
you get a voltage based estimate at system startup, it changes according to the measured current which gets integrated (mathematical) over time to give the accumulated used capacity and the voltage based estimate gets fused in the emptier the battery is to prevent you from deep discharging it because your battery doesn't actually have as much capacity as you configured

If you have a current sensor available the acumulated mAh capacity can always be seen in QGC for personal reference if you press on the battery icon even if it's not used for the battery estimation.